src/lib/evas/common/evas_thread_render.c

```diff
@@ -3,6 +3,10 @@
 #include "Ecore.h"
 #include <assert.h>
 
+#ifndef _MSC_VER
```

Comment:
Same as in this [one](https://github.com/expertisesolutions/efl/pull/159/files#r449040104)